crcls/blocksui-marketplace
# chore: update github action

## Purpose

[Preview](https://blocksui-5fg9gz3sb-crcls.vercel.app/)

1. Updated the GitHub Action to not deploy to Fleek since Vercel has its own action to deploy to preview environments and prod.
2. Added a type check command to the project, which runs locally on commit and in our GitHub Action.
3. There were some errors running the type check in GitHub’s CI, so the changes to the import paths and SVGs are related to that.

## Testing Instructions

1. Confirm there are no GitHub Action errors.
4. Confirm the SVGs do not appear broken in the [preview](https://blocksui-5fg9gz3sb-crcls.vercel.app/).

## Checklist
